src/uv-common.h

```diff
@@ -239,8 +239,12 @@ void uv__fs_scandir_cleanup(uv_fs_t* req);
 
 /* Allocator prototypes */
 void *uv__calloc(size_t count, size_t size);
+#if defined(__APPLE__)
 char *uv__strdup(const char* s);
+#endif
+#if 0
```

Comment:
wasn't sure why the declarations were left there so did't "dare" to remove them completely.  but will do so